cli/wizard/commands/list-squid-token/index.ts

```diff
@@ -115,69 +115,95 @@ export async function listSquidToken() {
   const tokenListPath = path.resolve(process.cwd(), ...relativePath);
 
   await spinner("Creating PR...", async () => {
-    await $`git checkout -b feat/add-${newTokenConfig.symbol}-token`;
+    await $`git checkout -b feat/add-${newTokenConfig.prettySymbol}-token`;
     await $`git add ${tokenListPath}`;
-    await $`git commit -m "feat: add ${newTokenConfig.symbol} token"`;
+    await $`git commit -m "feat: add ${newTokenConfig.prettySymbol} token"`;
     await $`git push -u origin HEAD`;
   });
 }
 
 export type InterchainTokenInfo = {
   tokenId: string;
   tokenAddress: string;
+  tokenManagerAddress: string;
+  tokenManagerType: string;
   isOriginToken: boolean;
   isRegistered: boolean;
   chainId: number;
   axelarChainId: string;
   chainName: string;
   wasDeployedByAccount?: boolean;
-  kind: "canonical" | "standardized";
+  kind: "canonical" | "interchain" | "customInterchain";
 };
 
 export type InterchainTokenSearchResult = InterchainTokenInfo & {
   matchingTokens: InterchainTokenInfo[];
 };
 
-export type RemoteInterchainToken = {
+export type RemoteInterchainTokenApiResponse = {
+  name: string;
+  symbol: string;
   axelarChainId: string;
   tokenAddress: string;
+  tokenManagerAddress: string;
+  tokenManagerType: string;
   deploymentStatus: string;
   deploymentTxHash: string;
 };
 
-export type InterchainTokenDetails = {
+export type InterchainTokenDetailsApiResponse = {
   kind: string;
   salt: string;
   tokenName: string;
   tokenSymbol: string;
   tokenDecimals: number;
   tokenAddress: string;
+  tokenManagerAddress: string;
+  tokenManagerType: string;
+  originalMinterAddress: string;
   axelarChainId: string;
   tokenId: string;
-  deploymentTxHash: string;
-  deployerAddress: string;
-  remoteTokens: RemoteInterchainToken[];
+  deploymentMessageId: string;
+  deployer: string;
+  remoteTokens: RemoteInterchainTokenApiResponse[];
 };
 
 function parseAsInterchainTokenConfig(
-  data: InterchainTokenDetails
+  data: InterchainTokenDetailsApiResponse
 ): InterchainTokenConfig {
   return {
     tokenId: hash.parse(data.tokenId),
-    tokenAddress: address.parse(data.tokenAddress),
-    symbol: data.tokenSymbol,
+    deployer: data.deployer,
+    originalMinter: data.originalMinterAddress,
     prettySymbol: data.tokenSymbol,
     decimals: data.tokenDecimals,
-    name: data.tokenName,
     originAxelarChainId: data.axelarChainId,
-    transferType: data.kind,
+    tokenType: data.kind,
+    deploySalt: data.salt,
     iconUrls: {
       svg: `${BASE_REPO_URL}/images/tokens/${data.tokenSymbol.toLowerCase()}.svg`,
     },
-    remoteTokens: data.remoteTokens.map((token) => ({
-      axelarChainId: token.axelarChainId,
-      tokenAddress: address.parse(token.tokenAddress),
-    })),
+    deploymentTxHash: data.deploymentMessageId?.split("-")[0] ?? "",
```

Comment:
I'm cool with just using deploymentMessageId in the config. Should generalize to non EVM chains better as well.